gov/politeia/piclient/piclient.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2019, The Decred developers
+// See LICENSE for details.
+
+// piclient handles the http requests made to API URL paths.
+
+package piclient
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// HandleGetRequests accepts a http client and API URL path as arguments. If the
+// parameters are valid, a GET request is made to the API URL path passed. The
+// body returned is then decoded into []byte and returned.
+func HandleGetRequests(client *http.Client, _URLPath string) ([]byte, error) {
```

Comment:
Variables with an underscore prefix are typically for reserved (e.g. language internals).  Unless we really need it this way, let's stick with `URLPath` or similar.